docs/guides/end-to-end-testing/migration/selenium-to-cypress.mdx

```diff
@@ -0,0 +1,431 @@
+and framework functionality. This analysis serves as a cornerstone for determining the priority of test case migration.
+Depending on the organization's needs, emphasis may be placed on high-priority cases rather than quick wins. Additionally,
+it is essential to evaluate whether any business or organization-specific logic embedded within the framework
+can be rebuilt or migrated to JavaScript or TypeScript.
+
+If the current test automation framework employs the Page Object Model, it may be advantageous to start with simpler
+tasks before tackling more complex test cases. Beginning with low-hanging fruit allows for the migration of selectors
+and basic interactions, providing the test automation engineer or developer with a gradual introduction to the framework.
+This incremental approach facilitates faster migration of additional tests and reduces the complexity of larger, more intricate tests.
+
+It's important to note that if the Selenium framework relies on XPath, migration will necessitate changing those selectors.
+While the cypress-xpath plugin remains functional, it is deprecated and may not be the optimal solution in the long term.
+
+Alternatively, if the current framework does not utilize the Page Object Model, selector migration will occur naturally as part of
+migrating the actual codebase. Alternatively, this could present an opportune moment to adopt the Page Object Model methodology,
+streamlining test maintenance and enhancing readability.
+
+## Getting started with Cypress
+
+To start with a new Cypress framework, open the CLI and navigate to the directory in which you wish to install Cypress.
+This can be inside the application directory, or you could consider Cypress as a separate entity. For the purpose of this guide,
+Cypress will be considered a separate framework, outside the application under test. There is no difference in installation or use.
+The reason this guide will handle it as a standalone application is because it makes screenshots clearer.
+
+<DocsImage
+  src="/img/guides/migrating-to-cypress/InstallCypress.png"
+  alt="install Cypress"
+/>
+
+<DocsImage
+  src="/img/guides/migrating-to-cypress/OpenCypressApp.png"
+  alt="open the Cypress app"
+/>
+
+<br />
+The first choice will be the testing type used by the framework, which in this migration
+guide will be E2E testing. After selecting an option, Cypress will inform you that
+it has created several directories and files.
+
+<DocsImage
+  src="/img/guides/migrating-to-cypress/WelcomeToCypress.png"
+  alt="welcome to Cypress screen"
+/>
+
+<br />
+<DocsImage
+  src="/img/guides/migrating-to-cypress/LookForCypressInstallFiles.png"
+  alt="look for Cypress install files and folders"
+/>
+
+<br />
+<br />
+Once all those files have been created, all that is left is selecting a browser
+to work against. After selecting a browser, Cypress will suggest creating an
+example spec, or you can jump right into it.
```

Comment:
```suggestion
to work against. After selecting a browser, Cypress will suggest creating
example specs, or you can jump right into it.
```

The user is offered "Scaffold example specs" (there are 20 of them) or "Create new spec" which is a single simple spec.